pkg/ratelimit/ratelimit.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ratelimit
+
+import (
+	"time"
+
+	"go.uber.org/ratelimit"
+)
+
+type Event struct {
+	Timestamp time.Time
+	Tag       int
+}
+
+type RateLimiter struct {
+	outputCh chan Event
+	C        <-chan Event
```

Comment:
What does `C` mean?

Reply:
`C` is just a public read-only view of `outputCh` so `RateLimiter` client could read events from it.
I have used `C` as name following the trail of `Timer`. I though it was a common practice in go, but not sure.

Reply:
I think this is fine. Maybe `C` is a bit too short, we can maybe rename it to `Events` for better clarity.